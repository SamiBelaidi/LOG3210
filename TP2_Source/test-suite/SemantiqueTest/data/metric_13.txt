Declaration
num a;
listnum [] b;
num c;
bool d;
listbool [] e;
EndDeclaration

a = 1 + 1;

foreach (num i: b) {
    a = a + (c * 1) + 5 / i;
    if(true) {
        c = 7;
    }
}

while ( 1 == 1 ) {
    a = 2 + 2 / 2;
    if ( (a + a < 5) && true) {
        d = (( true || false ) && true ) || false;

    } else {
        foreach(bool f : e) {
            f = a > 3 || 6 < 4 && 5 > 5;
            d = f;
        }
    }
    if (false) {
        foreach(num o : b) {
            f = a > 3 || 6 < 4 && 5 > 5;
            o = o;
        }
    }
}

