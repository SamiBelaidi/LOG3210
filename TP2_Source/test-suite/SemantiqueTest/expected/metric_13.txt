{VAR:8, WHILE:1, IF:3, FOR:3, OP:24}